### PR TITLE
Updated SEDMv2 reductions

### PR DESCRIPTION
**Pull request by saarahhall**
This PR adds dark reduction and sky background subtraction to the SEDMv2 pipeline (all configurations). Consequently, this changes the SEDMv2 test values and requires some new dark calibration files to be added to the mirar_starterpack.

### CODE REVIEW

File: mirar/pipelines/sedmv2/load_sedmv2_image.py [hunk 1]
```diff
@@ -172,7 +214,7 @@ def load_raw_sedmv2_mef(
     """
 
     sedmv2_ignore_files = [
-        "dark",
+        # "dark",
```
Comment (robertdstein) [inline mirar/pipelines/sedmv2/load_sedmv2_image.py:217]: Just remove?

File: mirar/pipelines/sedmv2/load_sedmv2_image.py [hunk 2]
```diff
@@ -197,10 +239,10 @@ def load_raw_sedmv2_mef(
             split_data = split_data[1:]
             split_headers = split_headers[1:]
         header, split_headers = clean_science_header(header, split_headers, is_mode0)
-    elif check_header["IMGTYPE"] in ["flat", "bias"]:
-        header, split_headers = clean_cal_header(header, split_headers[0], path)
+    elif check_header["IMGTYPE"] in ["flat", "bias", "dark", "unknown", "illum"]:
+        header, split_headers = clean_cal_header(header, split_headers, path, is_mode0)
     else:
-        logger.debug("Unexpected IMGTYPE. Is this a dark?")
+        logger.debug("Unexpected IMGTYPE.")
```
Comment (robertdstein) [inline mirar/pipelines/sedmv2/load_sedmv2_image.py:245]: Maybe should be `logger.debug("Unexpected IMGTYPE: {check_header['IMGTYPE']})` ?

**Review by robertdstein — CHANGES_REQUESTED** [review]

File: mirar/pipelines/sedmv2/load_sedmv2_image.py [hunk 3]
```diff
@@ -93,36 +93,77 @@ def clean_science_header(  # pylint: disable=too-many-branches
...
 
-    if not EXPTIME_KEY in informative_hdr:
-        header[EXPTIME_KEY] = informative_hdr[
-            "EXPOSURE"
-        ]  # be weary - exposure is not always reliable
+    if not is_mode0:
+        frame_time = 10  # pretty sure this is always true
+        # HUGE CHANGE - MEF is split up, so we need individual exptimes
+        if EXPTIME_KEY not in informative_hdr:
+            # be weary - keyword "EXPOSURE" is not always reliable
```
Comment (robertdstein) [inline mirar/pipelines/sedmv2/load_sedmv2_image.py:100]: be wary? though SEDmV2 does make you weary...

**Review by robertdstein — CHANGES_REQUESTED** [review]
Approved pending one minor typo

File: mirar/pipelines/sedmv2/load_sedmv2_image.py (hunk 3 above)
Comment (saarahhall) [reply mirar/pipelines/sedmv2/load_sedmv2_image.py:100]: haha

**Review by robertdstein — APPROVED** [review]

**Pull request merged**